docker-compose.dev.yml

```diff
@@ -32,6 +34,17 @@ services:
     volumes:
       - mongodb:/data/db
 
+  redis:
+    container_name: redis
+    image: redis:alpine
+    command: redis-server --requirepass sOmE_sEcUrE_pAsS
+    ports:
+      - 6379:6379
+    environment:
+      - REDIS_REPLICATION_MODE=master
+    volumes:
+      - redis:/data
+
```

Comment:
Is there a reason you did not add this to `docker-compose.production.yml` as well?